helpers/sendEmail.js

```diff
@@ -0,0 +1,19 @@
+const sgMail = require("@sendgrid/mail");
+require("dotenv").config();
+
+const { SENDGRID_API_KEY } = process.env;
+
+sgMail.setApiKey(SENDGRID_API_KEY);
+
+const sendEmail = async (data) => {
+  const email = { ...data, from: "marta.nikiel@yahoo.com" };
+
+  try {
+    await sgMail.send(email);
+    return true;
+  } catch (error) {
```

Comment:
```suggestion
  } catch (error) {
  console.log(error)
```
dobrze jeszcze mieć console log. W razie błędu

Reply:
ok